ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.scss

```diff
@@ -65,13 +67,11 @@
         display: flex;
         margin-bottom: 1em;
         padding-left: 10px;
-        i {
+        .nav-container {
+            display: flex;
+            padding: 3px 8px;
```

Comment:
```suggestion
        .page-info {
            margin-left: auto;
            margin-right: auto;
            padding: 3px 8px;
            text-overflow: clip;
            white-space: nowrap;
        }
        .nav-container {
            display: flex;
            align-self: flex-start; 
            padding: 3px 8px;
```

Reply:
Try this...replace lines 70-72, `.nav-container {` to `padding: 3px 8px;` with the above.

Reply:
Hey @rbreeze , your changes are fine.  I was just testing it out to see how the buttons look when you shrink the width of the browser.  This is what happens with and without my suggestion.  I guess it's just a matter of personal preference?  Feel free to disagree but I'll give it a LGTM.

With suggestion:
<img width="492" alt="WithSuggestion" src="https://user-images.githubusercontent.com/7726022/215233641-5e2ba37e-be6f-4eae-92bb-2473d556dce9.png">

As is:
<img width="390" alt="AsIs-WithoutSuggestion" src="https://user-images.githubusercontent.com/7726022/215233647-8b02015e-3727-4130-89f8-f357a10e5d64.png">

Reply:
Thanks for the review @keithchong! I agree, I like how it looks with your suggestion better.